Formula/m/macchina.rb

```diff
@@ -16,11 +16,26 @@ class Macchina < Formula
 
   depends_on "rust" => :build
 
+  resource "ansi-to-tui" do
+    url "https://github.com/Macchina-CLI/ansi-to-tui.git",
+        revision: "950d68067ed8c7f74469eb2fd996e04e1b931481"
+    sha256 "631ae65f367859fa1efade1656e4ba22b7da789c06e010cceb8b29656bf65757"
```

Comment:
You only need to specify one of `revision` or `sha256`. If you want to use `sha256` (preferred), just download the tarball from something like

https://github.com/Macchina-CLI/ansi-to-tui/archive/631ae65f367859fa1efade1656e4ba22b7da789c06e010cceb8b29656bf65757.tar.gz

FYI @daeho-ro

Reply:
Oh, I see. That's better.